pkg/openstack/rabbitmq.go

```diff
@@ -51,10 +96,26 @@ func ReconcileRabbitMQs(
 	var err error
 	var status mqStatus
 
+	Log := GetLogger(ctx)
+
 	if instance.Spec.Rabbitmq.Templates == nil {
 		instance.Spec.Rabbitmq.Templates = ptr.To(map[string]rabbitmqv1.RabbitMqSpecCore{})
 	}
 
+	toDeleteRabbitMQs, err := findToDeleteRabbitMQs(ctx, instance, helper)
+	if err != nil {
```

Comment:
Not sure if we should block when we have error in geting galeras to delete, plus it will be nice to try write some testcase